src/ConnectedMode/Migration/MigrationStrings.resx

```diff
@@ -117,6 +117,12 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="CancelTokenFailure_NormalLog" xml:space="preserve">
+    <value>[Migration] Migration cancelled: {0}</value>
```

Comment:
I would make this message `[Migration] Migration cancelled by user` - I don't think show the user the exception message will help, and might confuse the user.